games.html

```diff
@@ -33,6 +33,11 @@
     </head>
     
     <body>
+        <a href="#" class="back-to-top-arrow">
+            <svg xmlns="http://www.w3.org/2000/svg" width="23" height="23" viewBox="0 0 23 23" fill="none">
```

Comment:
Why is this SVG required?

Reply:
this svg is for the button that you click to go back to the top of the page.